test/memory.test.js

```diff
@@ -76,6 +85,35 @@ describe('Memory connector', function() {
       }, done);
     });
 
+    it('should not have out of sequence read/write', function(done) {
```

Comment:
Can you move this test at the end of the `describe` block, so that the existing tests depending on the exact number of `User` instances can remain unchanged?